Bindings/Laerdal.Maui.AzureCommunicationCalling.iOS/nativeLibs/Podfile

```diff
@@ -3,5 +3,5 @@ platform :ios, '14.0'
 use_frameworks!
 
 target 'nativeLibsPlaceholder' do
-   pod 'AzureCommunicationCalling', '~> 2.3.0-beta.1', :binary => true
+   pod 'AzureCommunicationCalling', '~> 2.8.0-beta.1', :binary => true
```

Comment:
Wait, this actually works? Is it published other places than github?

Reply:
`fetchPods.sh` will do the trick for you, but it doesn't have to be executed manually `create_binding.sh` will do everything for you